libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts

```diff
@@ -118,12 +118,12 @@ export class DhInviteUserModalComponent implements AfterViewInit, OnDestroy {
     email: [{ value: '', disabled: true }, Validators.required],
     phoneNumber: [
       '',
-      Validators.compose([
```

Comment:
Question: Why did you remove the compose?

Reply:
The required asterisk is only detected if the validation rule is directly present. Removing compose added the asterisk to phone number field.